<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>366</width>
    <height>545</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>366</width>
    <height>545</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>List of Games</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quick_math_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F44336;
color:white;
border-style: outset;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Quick Math</string>
    </property>
   </widget>
   <widget class="QPushButton" name="math_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FFA726;
color:white;
border-style: outset;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Math</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attention_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#FDD835;
color:white;
border-style: outset;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Attention</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reaction_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#8BC34A;
color:white;
border-style: outset;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Reaction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spatial_mem_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#B2EBF2;
color:white;
border-style: outset;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Spatial Mem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unscramble_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>430</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#3949AB;
color:white;
border-style: outset;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Unscramble</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>366</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
